dbt/models/model/model.vw_card_res_input.sql

```diff
@@ -654,6 +654,8 @@ LEFT JOIN (
     SELECT *
     FROM forward_fill
     WHERE NOT ind_pin_is_multicard
+        -- Some res parcels are not unique by pin10
+        AND SUBSTR(meta_pin, 11, 4) = '0000'
```

Comment:
**todo:** Can you add a little more context to the comment about why this is needed? Maybe just link to this issue.